source/clear-linux/tutorials/multi-boot/multi-boot.rst

```diff
@@ -111,99 +124,88 @@ Install the Clear Linux OS
 
 #. Start the |CL| installer and follow the prompts.
 
-#. On the *Choose Installation Type* screen, choose *Manual (Advanced)*,
-   as shown in Figure 1.
+#. On the :guilabel:`Choose Installation Type` screen, choose
+   :guilabel:`Manual (Advanced)`, as shown in Figure 1.
 
    .. figure:: figures/multi-boot-01.png
 
-      Figure 1: |CL| installer: Choose installation type screen
-
-   .. note::
-      If you are not familiar with text-based installation
-      interfaces, here are some navigation tips:
-
-      * Use the :kbd:`Up Arrow` and :kbd:`Down Arrow` keys to move between
-        the options on the screen.
-
-      * Use the :kbd:`Space` to select or highlight an option.
-
-      * Press :kbd:`Enter` to activate the selected option and to move ahead.
+      Figure 1: |CL| installer: Choose installation type screen.
 
 #. On the :guilabel:`Choose partitioning method` screen, choose
    :guilabel:`Manually configure mounts and partitions`, as shown in
    Figure 2.
 
    .. figure:: figures/multi-boot-02.png
 
-      Figure 2: |CL|: Choose partitioning method
+      Figure 2: |CL|: Choose partitioning method.
 
-#. Select the drive, in this case :file:`/dev/sda` and press :kbd:`Enter` to
+#. Select the drive, in this case :file:`/dev/sda`, and press :kbd:`Enter` to
    go into the `cgdisk` partitioning tool. See Figure 3.
 
    .. figure:: figures/multi-boot-03.png
 
-   Figure 3: |CL|: Choose drive to partition
+   Figure 3: |CL|: Choose drive to partition.
 
 #. Create a new root partition.
 
-   #. Select :guilabel:`New`. See Figure 4.
+   #. Select :guilabel:`New`, as shown in Figure 4.
 
       .. _multi-boot-04:
 
       .. figure:: figures/multi-boot-04.png
 
-         Figure 4: |CL|: Create new partition
+         Figure 4: |CL|: Create new partition.
 
    #. Accept the default first sector.
 
    #. Specify the desired size of the partition. For this example, we
-      specified *50GB*. See Figure 5.
+      specified *50 GB*. See Figure 5.
 
       .. figure:: figures/multi-boot-05.png
 
-         Figure 5: |CL|: New partition size
+         Figure 5: |CL|: New partition size.
 
-   #. Set the partition type to :guilabel:`8300 (Linux filesystem)`. See
-      Figure 6.
+   #. Set the partition type to :guilabel:`8300 (Linux filesystem)`, as shown
+      in Figure 6.
 
       .. figure:: figures/multi-boot-06.png
 
-         Figure 6: |CL|: Set partition type
+         Figure 6: |CL|: Set partition type.
 
    #. Name the partition :file:`CL-root`. This name makes it easier to
       identify later. See Figure 7.
 
       .. figure:: figures/multi-boot-07.png
 
-         Figure 7: |CL|: Name partition
+         Figure 7: |CL|: Name partition.
 
-#. Create a new swap partition. See Figure 8.
+#. Create a new swap partition as shown in Figure 8.
```

Comment:
@MCamp859 - we're been told to use "figure" (lower case) instead of "Figure" when referenced in a sentence.  Maybe you can change all of them?

Reply:
Deferred. AR Mary to research doc guidelines and respond in WW22 after vacation.